File: official/docs/php/current/batches/remove-shipments.php
<?php

$client = new \EasyPost\EasyPostClient('EASYPOST_API_KEY');

$batch = $client->batch->removeShipments(
    'batch_...',
    [
        'shipments' => [
            ['id' => 'shp_...']
        ]
    ]
);

echo $batch;

File: official/docs/python/v7/batches/remove-shipments.py
import easypost
import os

easypost.api_key = os.getenv("EASYPOST_API_KEY")

batch = easypost.Batch.retrieve("batch_...")

batch.remove_shipments(
    shipments=[
        {
            "id": "shp_...",
        }
    ]
)

print(batch)
